sbin/system-setup.py

```diff
@@ -40,15 +40,22 @@ def debian_compat(self):
                 self.install("libffi-dev")
 
     def redhat_compat(self):
-        self.install("redhat-lsb-core")
+        if self.dist == "centos" and self.os_version[0] < 9:
+            self.install("redhat-lsb-core")
+
         self.install("which")
-        self.run(f"{READIES}/bin/getepel", sudo=True)
+        self.run(f"{READIES}/bin/getepel")
         self.install("libatomic")
 
         if self.dist == "centos" and self.os_version[0] == 7:
             self.run(f"{READIES}/bin/getgcc --modern --update-libstdc++")
+        elif self.dist == "centos" and self.os_version[0] == 9:
+            # avoid gcc 12 for the time being
+            self.run(f"{READIES}/bin/getgcc")
         else:
             self.run(f"{READIES}/bin/getgcc --modern")
+        self.install("libstdc++-static")
```

Comment:
What is that for?

Reply:
@alonre24 We link statically to libstdc++ (to avoid runtime dependency in platforms with an older toolchain), and this package is required for that to be available with the toolchain (it was once available by default and probably dropped on newer versions of gcc on some platforms, e.g. rocky9).